docs/data/data-grid/server-side-data/index.md

```diff
@@ -189,6 +189,43 @@ This means that if the user navigates to a page or expands a node that has alrea
 
 The `GridDataSourceCacheDefault` is used by default which is a simple in-memory cache that stores the data in a plain object. It can be seen in action in the [demo above](#with-data-source).
 
+### Improving the cache-hit ratio
```

Comment:
This sounds to me a nice candidate for an FAQ section, wdyt about adding a FAQ page under "Server side data" section and add this as the first FAQ?

Reply:
I don't mind. Either that or adding it to the Data Grid FAQ
we don't have FAQ pages per feature yet
@samuelsycamore what would be your advice?

Reply:
I remember having a discussion where it was proposed to have per-section FAQs where it makes sense to avoid polluting the main FAQ with feature-specific details. Anyhow, @samuelsycamore's take would be helpful here. 👍

Reply:
I'm not a fan of FAQ pages for this kind of content—these pages often feel random and disorganized, and they become a catch-all for content that hasn't found a more specific home. If there's a common use case that needs to be documented, then there's probably a better location for it within existing documentation already, or else we should consider creating a new page. 

In this case, it seems to me like this material is already in the right place: immediately following the section where it would be the most relevant. If this is a common situation that users face, then we should solve the problem on the same page where we introduce the feature so they don't have to navigate to a different page.

All that said, this is becoming a pretty massive document. Maybe there's a better way to subdivide the info by splitting some of the content off into new, more specific pages?

Reply:
I have added an [issue](https://github.com/mui/mui-x/issues/15638) to address this
I think that caching could be its own page at this point
@samuelsycamore feel free to add your ideas to the issue